### DISCUSSION TITLE
Bcrypt.dll

### DISCUSSION THREAD

**Discussion by skoozeblood** [OwlGamingCommunity/MTA#42] [General]
Error: account/login-panel/server.lua:372: attempt to call global 'bcrypt_gensalt' (a nil value) #7

@ChaosParadise deleted an apparent workaround of this problem made by @PhatChilla so if that's not allowed what's the solution?

**Comment by saviorsoldier** [comment]
Use docker file on Linux server or get the necessarry files by downloading Windows Redist.
